.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,3 +25,4 @@
 - [ ] Check that new functions are imported in corresponding `__init__.py`.
 - [ ] Check that new features, API changes, and deprecations are mentioned in
       `doc/release/release_dev.rst`.
+- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`
```

Comment:
I'd change `v0.14.x` to something like `{branch_name}`

Reply:
We are only supporting v0.14.x right now. This is really meant to make it super easy for you guys to copy paste the command when you click "merge".